talend-component-maven-plugin/src/it/dependency/postbuild.groovy

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
The class `ValidateDependencies` seems not to be run.
I could not find anything in the invoker plugin's documentation hinting at Java classes from the IT projects being run.
I guess they should since the class is compiled and the `default-test` phase is run but I updated the class to throw and write crap to the FS to check whether it was run and it seems not.

I didn't fix it since this PR is reaaaaaaaaally dragging already but I'm almost positive that it's not run and thus useless.

Reply:
The test is not run because the pom lacks the failsafe plugin execution and test file is not following IT naming.
Anyway, not sure that this test is really helpful: Had a quick look at it not sure that it passes ...